dev-libs/libdivecomputer/libdivecomputer-0.6.0.ebuild

```diff
@@ -0,0 +1,34 @@
+# Copyright 1999-2016 Gentoo Foundation
```

Comment:
Looks good, but please update these copyright headers. If you commit using repoman, it fixes them for you automatically (and adds sign-off automatically as well)